uui/components/filters/FiltersPanelItem.tsx

```diff
@@ -153,6 +155,18 @@ function FiltersToolbarItemImpl(props: FiltersToolbarItemProps) {
         return predicate ? props.value?.[predicate] : props.value;
     };
 
+    const getName = (item: DataRowProps<any, any>, footerProps: PickerFilterConfig<any>) => {
```

Comment:
Do we need here second param, can we use props from closure? Also we get name only for pickers, let's name it more specific, for example - getPickerItemName()

Reply:
Done. But we can't take props from closure, because `switch` in this case is playing the role of the guard, which determines the exact type of props from the union of types.